tests/test_indieauth/test_verification.py

```diff
@@ -0,0 +1,57 @@
+from indieweb_utils.indieauth import server
+
+
+class TestGenerateAuthToken:
+    def test_generate_auth_token(self):
+        """Test function to generate an authentication token."""
```

Comment:
Might be better to add some more explanation of the test in the doc strings.
```suggestion
        """A end-to-end happy path test to confirm the entire indieauth flow is working properly.
         It will: generate an auth token, redeem the token, validate access, and validate authorization.
        """
```